Zend/zend_portability.h

```diff
@@ -575,14 +576,15 @@ static zend_always_inline double _zend_get_nan(void) /* {{{ */
 # define ZEND_INTRIN_SSSE3_FUNC_DECL(func)
 #endif
 
-#if defined(HAVE_SSE4_2_DEF) || (defined(ZEND_WIN32) && defined(__SSE4_2__))
+#if __SSE4_2__
```

Comment:
`#ifdef __SSE4_2__`
was this line before #3826 (but not sure if important)

Reply:
Thanks for the pointer, I'll change this. It makes no difference here, but better to stay consistent with the other checks.